lib/dialects/abstract/query-generator.js

```diff
@@ -1095,12 +1095,13 @@ class QueryGenerator {
   /**
    * Generates an SQL query that extract JSON property of given path.
    *
-   * @param   {string}               column  The JSON column
-   * @param   {string|Array<string>} [path]  The path to extract (optional)
-   * @returns {string}                       The generated sql query
+   * @param   {string}               column   The JSON column
+   * @param   {string|Array<string>} [path]   The path to extract (optional)
+   * @param   {boolean}              [isJson] The value is JSON use alt symbols (optional)
+   * @returns {string}                        The generated sql query
    * @private
    */
-  jsonPathExtractionQuery(column, path) {
+  jsonPathExtractionQuery(column, path, isJson) {
```

Comment:
I think we should rather have a third Paramus which allows to override the joint fragment. What do you think?